src/config_models/cli_args.rs

```diff
@@ -170,6 +170,21 @@ pub struct Args {
     #[clap(long)]
     pub(crate) max_mempool_num_tx: Option<usize>,
 
+    /// Restrict the number of inputs for transcations that this node deals
+    /// with.
+    ///
+    /// If this value is set, the node will not create transactions with more
+    /// inputs than this limit. It will also not accept such transactions from
+    /// peers into its mempool. And if the node is either upgrading transaction
+    /// proofs on the network or composing, it will never merge two transactions
+    /// that would exceed this limit. Does not affect block validity, i.e.
+    /// consensus. So a block with more than this number of inputs may be
+    /// considered value. Also does not affect which block proposals that a
+    /// guessing node will attempt to solve the PoW puzzle for. Set to "none" to
+    /// disable this limit.
```

Comment:
If the default is not none, it should say here what it is.

Reply:
`clap` fills that out automatically when you use e.g. `neptune-core --help`. See this example for how it works with `--max-num-proofs`:
```bash
      --max-num-proofs <MAX_NUM_PROOFS>
          Sets the maximum number of proofs in a `ProofCollection` that can be recursively combined into a `SingleProof` by this machine. I.e. how big STARK proofs this machine can produce
          
          [default: 16]

```